csharp_webapi/11.QnAMaker/appsettings.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "MicrosoftAppId": "",
```

Comment:
appsettings.json should not be checked in. Also the botFilePath and botFileSecret should be in appsettings.json